beetmoverscript/constants.py

```diff
@@ -155,6 +158,11 @@
     'thunderbird': 'pub/thunderbird/',
 }
 
-PARTNER_LEADING_STRING = 'releng/partner/'
-PARTNER_REPACK_PUBLIC_PREFIX_TMPL = 'pub/firefox/candidates/{version}-candidates/build{build_number}/partner-repacks/'
-PARTNER_REPACK_PUBLIC_PAYLOAD_ID = 'is_partner_repack_public'
+PARTNER_REPACK_PUBLIC_PREFIX_TMPL = 'pub/firefox/candidates/{version}-candidates/build{build_number}/'
+PARTNER_REPACK_PRIVATE_REGEXES = (
+    r'^[^\/.]+/{version}-{build_number}/[^\/.]+\/(mac|win32|win64|linux-i686|linux-x86_64)/[^\/.]+$',
```

Comment:
The `.../[^\/.]+\/...` part in the middle of PARTNER_REPACK_PUBLIC_REGEXES doesn't like partner subconfigs like mail.com_notb in unitedinternet ([eg](https://tools.taskcluster.net/groups/I_lv8okdSXqL-1J6o_xgGQ/tasks/dhSha55MQhOGwFet-f8dkA/details)). Maybe replace it with a `.../[^\/]+?/...`  (ie non-greedy matching allowing periods, and remove what looks like a stray escape the following /).

Reply:
I added `.` to disallow `/../` patterns. I can try to change the regex and checks to allow for a normal usage of periods, but not `..`